tests/extensions/default/Microsoft.DotNet.UpgradeAssistant.Steps.Backup.Tests/BackupStepTests.cs

```diff
@@ -144,6 +145,7 @@ private static IUpgradeContext GetContext(AutoMock mock, string inputPath, strin
             context.Setup(c => c.InputPath).Returns(inputPath);
             context.Setup(c => c.InputIsSolution).Returns(Path.GetExtension(inputPath).Equals(".sln", StringComparison.OrdinalIgnoreCase));
             context.Setup(c => c.Properties).Returns(mock.Container.Resolve<IUpgradeContextProperties>());
+            context.Setup(c => c.Results).Returns(new Mock<ICollector<OutputResultDefinition>>().Object);
 
```

Comment:
Could we also add tests that check to make sure Results are populated as we anticipate w/ the other changes in this PR?

Reply:
We should be able to test this by fixing the broken integration tests (which should now expect the upgrade report)